src/components/common/NameRequestInfo.vue

```diff
@@ -156,9 +156,11 @@ export default class NameRequestInfo extends Mixins(CommonMixin, DateMixin, Enum
 
   #name-request-summary {
     font-size: 0.9rem;
+    margin-left: 0.875rem
   }
 
   .nr-status-icon {
-    margin-left:-2rem
+    margin-left:-2rem;
+    margin-right:0.2 rem
```

Comment:
In the future, keep an eye on spacing and punctuation.
- normally CSS properties should end with a semi-colon
- normally there should be a space between colon and value
- normally there is no space between value and "rem"... does it actually work here?